### PR TITLE
Reduce flake8 max-line-length

### PR DESCRIPTION
**Pull request by mercybassey**
# Description

Reduced flake8 maximum line length 

# Changed Behaviour

Which existing user workflows or functionality will behave differently after this PR?

# Fixes

Fixes # 3105

## Type of change

Choose which options apply, and delete the ones which do not apply.

- Code maintenance/cleanup

### CODE REVIEW

File: parsl/monitoring/db_manager.py [hunk 1]
```diff
@@ -544,7 +544,11 @@ def start(self,
                 if reprocessable_last_resource_messages:
                     self._insert(table=STATUS, messages=reprocessable_last_resource_messages)
             except Exception:
-                logger.exception("Exception in db loop: this might have been a malformed message, or some other error. monitoring data may have been lost")
+                log_message = (
```
Comment (benclifford) [inline parsl/monitoring/db_manager.py:547]: Hi, thanks for your PR. You could avoid using an intermediate variable here. You could use multi-line string syntax directly on the method call to logger.exception. Here's an example function call like that to show what I mean:

```
def f(x: str):
 print(x)

f('a'
  'b')
```

**Comment by benclifford** [comment]
It looks like I'll probably merge PR #3129 because it will be ready slightly before yours. But this PR is fine otherwise. If you want, you can wait a few minutes and reduce the max line length even further, or you can move on to looking at some of the other outreachy issues - I'm going to tag some more now.

**Comment by mercybassey** [comment]
> It looks like I'll probably merge PR #3129 because it will be ready slightly before yours. But this PR is fine otherwise. If you want, you can wait a few minutes and reduce the max line length even further, or you can move on to looking at some of the other outreachy issues - I'm going to tag some more now.

Alright, I'll wait a few minutes and reduce the max line length even further. However, I'll also move on to looking at some of the other outreachy issues too.

**Review by benclifford — APPROVED** [review]

**Comment by benclifford** [comment]
turns out this PR got merged after all and #3129 didn't - just because the fixes and tests passed a few minutes before the #3129 changes (which look basically the same)

**Pull request merged**